stuff/javascript/jasmine-standalone-2.8.0/src/getExponent.js

```diff
@@ -0,0 +1,8 @@
+function getExponent(a, b) {
+	var res = Math.pow(a, b);
+
+	if (isNaN(res)) 
+		throw Error('invalid conditions');
+
+    return res;
+}
```

Comment:
The message "No new line at end of file" is not a critical problem. But please, check this link ->

https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file